pl/mozorg/contribute/index.lang

```diff
@@ -73,16 +73,16 @@ Share your skills and pick up<br> a few new ones along the way.
 
 
 ;Volunteering with Mozilla&hellip;
-Wolontariat dla Mozilli&hellip;
+Wolontariat w Mozilli&hellip;
 
 
 # This paragraph follows the headline, "Volunteering with Mozilla...", so the first sentence continues from that fragment. The line breaks are for formatting.
 ;is about more than lending a hand. It’s about learning, collaborating and sharing your skills with a global community of Mozillians — coders, organizers, activists and more — who help shape the Web every day.
-is about more than lending a hand. It’s about learning, collaborating and sharing your skills with a global community of Mozillians — coders, organizers, activists and more — who help shape the Web every day.
+to więcej niż podanie ręki. Pozwala nauczyć się, współpracować i dzielić się swoimi umiejętnościami z globalną społecznością Mozillian — programistów, organizatorów, aktywistów i więcej - pomagających zmieniać kształt sieci każdego dnia.
```

Comment:
Lepiej mi brzmi „uczyć się” niż „nauczyć się”.
„więcej” → „i nie tylko”
„sieci” → „Internetu”
Druga kreska to myślnik zamiast em-dash.